### PR TITLE
DEVX-1880: Modify double schema.registry C3 config

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -607,9 +607,9 @@ services:
       CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL: http://localhost:8088
 
       # Schema Registry
-      CONTROL_CENTER_SCHEMA_REGISTRY_URL: https://schemaregistry:8085
-      CONTROL_CENTER_SCHEMA_REGISTRY_SCHEMA_REGISTRY_SSL_TRUSTSTORE_LOCATION: /etc/kafka/secrets/kafka.client.truststore.jks
-      CONTROL_CENTER_SCHEMA_REGISTRY_SCHEMA_REGISTRY_SSL_TRUSTSTORE_PASSWORD: confluent
+      CONTROL_CENTER_SCHEMA_REGISTRY_SR1_URL: https://schemaregistry:8085
+      CONTROL_CENTER_SCHEMA_REGISTRY_SR1_SCHEMA_REGISTRY_SSL_TRUSTSTORE_LOCATION: /etc/kafka/secrets/kafka.client.truststore.jks
+      CONTROL_CENTER_SCHEMA_REGISTRY_SR1_SCHEMA_REGISTRY_SSL_TRUSTSTORE_PASSWORD: confluent
```
Comment (nicolasguyomar) [inline docker-compose.yml:612]:
```suggestion
      CONTROL_CENTER_SCHEMA_REGISTRY_SR1_SCHEMA_REGISTRY_URL: https://schemaregistry:8085
      CONTROL_CENTER_SCHEMA_REGISTRY_SR1_SSL_TRUSTSTORE_LOCATION: /etc/kafka/secrets/kafka.client.truststore.jks
      CONTROL_CENTER_SCHEMA_REGISTRY_SR1_SSL_TRUSTSTORE_PASSWORD: confluent
```

Comment (nicolasguyomar) [reply docker-compose.yml:612]: I believe you have to have the CONTROL_CENTER_SCHEMA_REGISTRY previx, then the SR cluster name, then the properties
and SR client properties for the SR Url is "schema.registry.url"

Comment (ybyzek) [reply docker-compose.yml:612]: @nicolasguyomar 

1. Did you test the suggested change?
2. Docs say: "Multiple Schema Registry clusters may be specified with confluent.controlcenter.schema.registry.{name}.url fields." It uses `confluent.controlcenter.schema.registry.{name}.url`, not `confluent.controlcenter.schema.registry.{name}.schema.registry.url`

cc: @emikjackson to clarify

Comment (mikebin) [reply docker-compose.yml:612]: @ybyzek - your property names look good to me according to https://github.com/confluentinc/blueway/blob/5.5.0-post/control-center/src/main/java/io/confluent/controlcenter/schemaregistry/SchemaRegistryClusterMetadata.java

Comment (emikjackson) [reply docker-compose.yml:612]: Hi @ybyzek, your original config and understanding is correct.

**Pull request merged**